modules/phase_field/test/tests/grain_growth/bad_text

```diff
@@ -0,0 +1,6847 @@
+[33mtest:grain_growth.voronoi: [0mWorking Directory: /Users/kongf/projects/moose4/modules/phase_field/test/tests/grain_growth
```

Comment:
Did you mean to check in these two outputs?

Reply:
Haha, git added everything